core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -304,7 +304,7 @@ private void parseArguments(final String[] args) {
             optionParser,
             options,
             option,
-            JavaConverters.asScalaSetConverter(invalidOptions).asScala());
+            CollectionConverters.SetHasAsScala(invalidOptions).asScala());
```

Comment:
Seems to be a result of KIP-751 (drop support for Scala 0.12) ?

Reply:
This is deprecated by scala (scala 2.12, not 0.12 :)). Here's the note from scala doc in 2.12.0, to explain why it is deprecated:
>  The transparent conversions provided here are considered fragile because they can result in unexpected behavior and performance.

> Therefore, this API has been deprecated and JavaConverters should be used instead.

I think that time, `CollectionConverters` haven't existed, so suggest to use `JavaConverters` instead. 
REF: https://www.scala-lang.org/api/2.12.0/scala/collection/JavaConversions$.html

Reply:
Sorry, I was wrong. Have a 2nd look, and found `CollectionConverters` doesn't exist in scala 2.12. So, this change will not be compatible to scala 2.12. This change should be reverted. Thanks.